tests/test_get_episodes.py

```diff
@@ -1,45 +1,77 @@
 import unittest
 import requests
+
+
 BASE_URL = 'https://www.breakingbadapi.com/api/'
+HTTP_OK = 200
+TOTTAL_EPISODES = 102
 
 
 class TestEpisodes(unittest.TestCase):
 
+    def _check_episodes_attr(self, episode):
```

Comment:
nit: This can be refactored to be more 'pythonic' and extendable. For example:

`def _check_required_atributes_are_present(self, episode) -> None:
    required_attributes = (
        'episode_id',
        'title',
        'season',
    )
    are_required_attributes_present: bool = all([attr in episode for attr in required_attributes])
    self.assertTrue(are_required_attributes_present, 'Missing some of the required attributes')
`
`